demo/Phonebook.MockBackend.yml

```diff
@@ -54,5 +54,7 @@ spec:
         paths:
           - path: /api
             backend:
```

Comment:
Is no `pathType` necessary, as explained in https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types `Paths that do not include an explicit pathType will fail validation. ` ?

Reply:
yes! good catch. 
I should have just deleted it as it seems to work without. But thats for another time.